header-footer.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="sv-se">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
is this a testing file just for header and footer to be visualized? if it is possible, header should be in the header.html and footer should be in the footer.html. You can find header.html and footer.html in a folder called 'common'